pkg/api/filters/headers/validator.go

```diff
@@ -0,0 +1,56 @@
+package headers
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/Azure/open-service-broker-azure/pkg/api/filters"
+	log "github.com/Sirupsen/logrus"
+)
+
+var responseMissingAPIVersion = []byte(
+	`{ "error": "MissingAPIVersion", "description": "The request did not ` +
+		`include the X-Broker-API-Version header"}`,
+)
+
+var responseAPIVersionIncorrect = []byte(
+	`{ "error": "APIVersionIncorrect", "description": "X-Broker-API-Verson ` +
+		`header includes an incompatible version"}`,
+)
+
+type validator struct{}
```

Comment:
Should we be more specific about _which_ header this validates? There are conceivably other headers we might wish to validate in the future. Or maybe we could make this one generic, and pluggable with a "constructor" that takes a header name and a `func(value string) bool`? Thoughts?

Reply:
Yeah, I think the more generic form with a pluggable constructor is better. It's probable that we will need to fix this later.